GCMessengerSDKSample/GCMessengerSDKSample/ChatWrapperViewController.swift

```diff
@@ -101,6 +101,12 @@ extension ChatWrapperViewController: ChatControllerDelegate {
                 }
                 stopSpinner(activityView: chatViewControllerActivityView)
                 
+            case .failedToAutostartConversation:
+                print("** Error: \(error.errorType.rawValue)")
+                if let errorDescription = error.errorDescription {
+                    Toast.show(message: errorDescription)
```

Comment:
I don't think we should toast this message on the sample.
User will no about the failure according to the welcome message, just keep the log

Reply:
Because we use the sample for testing purposes and not for the general audience, I feel like having a toast will benefit us for now